test/smiley_examples.jl

```diff
@@ -0,0 +1,220 @@
+    prod(as * x[1] .+ bs * x[2] .+ cs * x[3] .- ds)
+)
+
+# contains all 24 reported roots
+const region = IntervalBox(-3..3, -3..3, -3..3)
+
+const known_roots = [
+    IntervalBox(-1.933009 ± 1e-6, -0.300000 ± 1e-6,  0.416504 ± 1e-6),
+    IntervalBox(-1.701684 ± 1e-6,  0.000000 ± 1e-6,  1.050842 ± 1e-6),
+    IntervalBox(-1.258803 ± 1e-6,  1.360696 ± 1e-6, -0.750946 ± 1e-6),
+    IntervalBox(-1.044691 ± 1e-6, -1.589843 ± 1e-6,  0.617267 ± 1e-6),
+    IntervalBox(-0.996600 ± 1e-6,  1.726162 ± 1e-6, -0.164780 ± 1e-6),
+
+    IntervalBox(-0.951026 ± 1e-6, -0.300000 ± 1e-6, -0.074486 ± 1e-6),
+    IntervalBox(-0.800000 ± 1e-6, -0.000000 ± 1e-6,  0.600000 ± 1e-6),
+    IntervalBox(-0.776373 ± 1e-6, -1.344718 ± 1e-6,  1.260546 ± 1e-6),
+    IntervalBox(-0.717665 ± 1e-6,  0.423418 ± 1e-6, -0.552876 ± 1e-6),
+    IntervalBox(-0.592072 ± 1e-6, -0.805884 ± 1e-6, -0.001021 ± 1e-6),
+
+    IntervalBox(-0.499927 ± 1e-6,  0.865900 ± 1e-6,  0.017013 ± 1e-6),
+    IntervalBox(-0.360640 ± 1e-6, -0.624646 ± 1e-6,  0.692643 ± 1e-6),
+    IntervalBox( 0.082249 ± 1e-6, -0.962075 ± 1e-6, -0.260086 ± 1e-6),
+    IntervalBox( 0.085220 ± 1e-6,  0.367221 ± 1e-6, -0.926221 ± 1e-6),
+    IntervalBox( 0.453788 ± 1e-6,  0.785984 ± 1e-6, -0.419886 ± 1e-6),
+
+    IntervalBox( 0.464511 ± 1e-6, -0.804557 ± 1e-6,  0.370022 ± 1e-6),
+    IntervalBox( 0.511026 ± 1e-6, -0.300000 ± 1e-6, -0.805513 ± 1e-6),
+    # the following two roots are suspect, first column probably reported in error
+    #IntervalBox( 0.623386 ± 1e-6,  1.151180 ± 1e-6, -1.544510 ± 1e-6),
+    #IntervalBox( 0.869521 ± 1e-6, -1.899353 ± 1e-6, -0.062016 ± 1e-6),
+    IntervalBox( 0.537839 ± 1e-6,  1.151180 ± 1e-6, -1.544510 ± 1e-6),
+    IntervalBox( 0.623386 ± 1e-6, -1.899353 ± 1e-6, -0.062016 ± 1e-6),
+    IntervalBox( 0.869521 ± 1e-6,  1.506056 ± 1e-6, -0.987788 ± 1e-6),
+
+    IntervalBox( 0.960000 ± 1e-6,  0.000000 ± 1e-6, -0.280000 ± 1e-6),
+    IntervalBox( 0.961183 ± 1e-6, -1.664819 ± 1e-6,  0.551817 ± 1e-6),
+    IntervalBox( 1.493009 ± 1e-6, -0.300000 ± 1e-6, -1.296504 ± 1e-6),
+    IntervalBox( 1.861684 ± 1e-6,  0.000000 ± 1e-6, -0.730842 ± 1e-6),
+]
+
+end
+
+# example 5.4, rescaled form
+module SmileyExample54
+
+using IntervalArithmetic
+using StaticArrays
+
+const title = "Smiley and Chun (2001), Example 5.4"
+
+const c11 =  1.069e-5
+const c12 =  2e2
+const c13 =  1e5
+const c14 = -1.8e5
+const c15 = -1.283e-4
+const c21 =  2e-2
+const c22 =  1e1
+const c23 = -1e1
+
+f(t) = SVector(
+    c11 * t[1]^4 + c12 * t[1]^3 * t[2] + c13 * t[1]^3 + c14 * t[1] + c15,
+    c21 * t[1] * t[2]^2 + c22 * t[2]^2 + c23
+    )
+
+# contains all 7 reported roots
+const region = IntervalBox(-5.1e2..1.4, -5.1e2..1.1)
+
+const known_roots = [
+    IntervalBox(-1.34298 ± 1e-5, -1.00134 ± 1e-5),
+    IntervalBox(-1.34030 ± 1e-5,  1.00134 ± 1e-5),
+    IntervalBox( 1.34030 ± 1e-5,  0.99866 ± 1e-5),
+    IntervalBox( 1.34298 ± 1e-5, -0.99866 ± 1e-5),
+    # following three roots are not reported accurately
```

Comment:
Not reported accurately by the package or the original paper?

Reply:
in the original paper. with the reference values i tried to stick as closely as possible to the reported data.

Reply:
maybe this should say "not reported precisely" because the reported values are broadly correct, just not very precise.